bindata/network/ovn-kubernetes/004-config.yaml

```diff
@@ -30,7 +30,8 @@ data:
     [gateway]
     mode={{.OVN_GATEWAY_MODE}}
     nodeport=true
-{{ if .OVNHybridOverlayEnable }}
+{{- if .OVNHybridOverlayEnable }}
+
```

Comment:
why the extra line, it doesn't work without this?

Reply:
It's only for cosmetic. I just wanted to have like for the others, e.g. `[kubernetes]` has an empty line above as well. If I would have added the empty line before the `if`, it would might end up in multiple empty lines rendered.

Reply:
ack, my eyes just got caught on the unrelated change on that line.